test/test-Consumer.js

```diff
@@ -681,10 +681,7 @@ test('consumer.dump() succeeds', async () =>
 				{
 					ssrc : videoConsumer.rtpParameters.encodings[0].rtx.ssrc
 				},
-				scalabilityMode : 'S4T1',
-				spatialLayers   : 4,
-				temporalLayers  : 1,
-				ksvc            : false
```

Comment:
Why are these fields removed? Actually they represent real fields that must be set.

Reply:
The reason is that these are inconsistent with defined types.
https://mediasoup.org/documentation/v3/mediasoup/rtp-parameters-and-capabilities/#RtpParameters doesn't have `spatialLayers`, `temporalLayers` and `ksvc` in it and thus identical test in Rust wasn't possible to write because it just wouldn't compile.

Since this is a dump and the only place where I found these properties are used were tests, I decided to remove them from tests and from corresponding place in the worker itself. At the end of the day `scalabilityMode` already encapsulates all three and you can parse it into those fields if really needed.

Reply:
It's not a dump, these fields are part of "our" `RtpEncodingParameters` type. We may have to properly document them indeed, but they are needed. In fact we use them in production.

BTW `scalabilityMode` is a real W3 field defined in an additional spec: https://www.w3.org/TR/webrtc-svc/

Reply:
Hm... this particular `data` is a dump as you can see from the call above and `data.rtpParameters` indeed comes from `RtpEncodingParameters::FillJson` in the worker, which is an implementation for dumping said object.

Neither documentation nor TypeScript code has `spatialLayers`, `temporalLayers` or `ksvc` here.

I did NOT remove `scalabilityMode` (which I think is the source of confusion here). It is red on this side of the diff, but it is also green on the other side. Is that your concern here?